src/main/java/com/microsoft/sqlserver/jdbc/SQLServerPreparedStatement.java

```diff
@@ -253,6 +256,7 @@ private boolean resetPrepStmtHandle(boolean discardCurrentCacheItem) {
         procedureName = parsedSQL.procedureName;
         bReturnValueSyntax = parsedSQL.bReturnValueSyntax;
         userSQL = parsedSQL.processedSQL;
+        userSQLUnprocessed = sql;
```

Comment:
if we do the check here then we won't need to keep a copy of the sql command

Reply:
I was thinking for the future where we want to do it the "jTDS" way, we'd need to track the unprocessed sql for the SQL Batch TDS message. But I'll switch to boolean for now.